Week3/homework/creditCardValidator.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+const validateCreditCard = (creditCardNum) => {
+
+    // The credit card number must be 16 digits in length
+    if (creditCardNum.length !== 16) {
+        return false;
+    }
+
+    // All of the digits must be numbers
+    for (let i = 0; i < creditCardNum.length; i++) {
+        // store the current digit
+        let currentNumber = creditCardNum[i];
+
+        // turn the digit from a string to an integer (if the digit is in fact a digit and not anther char)
+        currentNumber = Number.parseInt(currentNumber);
+
+        // check that the digit is a number
+        if (!Number.isInteger(currentNumber)) {
+            return false;
+        }
+    }
```

Comment:
Your logic works which is great.
Another way to do this more compactly is using `Number()` object, see: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number
It tries to parse the entire string and see if it is a number, and if not returns `NaN`.

You could do a check like `Number.isInteger(Number(creditCardNum))`